### ISSUE TITLE
TabMon Fails to run "The system cannot find the file specified"

### ISSUE THREAD

**Issue by FLuGonz** [tableau/TabMon#200]
Hi! We have currentlz a problem when running tabmon with the StartTabMon.bat batch file. Every time this task is supposed to run, we get the following message:

System error 2. The system cannot find the file specified.

Could you please let us know how can we fix this issue? Thanks!

**Comment by danjrahm** [comment]
Hello @FLuGonz,

To figure out what is happening lets perform the following steps:

**Remove all existing TabMon logs:**
1. Navigate to the TabMon Logs directory <Root>\TabMon\Logs
2. Delete all text files there.

**Run TabMon In Command Prompt**
1. Open the Command Prompt as an administrator.
2. Navigate to the TabMon directory.
3. Run TabMon.exe

Once done, this should give you the exact failure from the command as well as the logs with the errors. After performing the steps please send me both the logs and the command output.

Thanks,
Dan

**Comment by FLuGonz** [comment]
Hi @danjrahm,

thanks for your help. I have managed to troubleshoot the problem in the end. We had 2 different issues. First of all we were getting a timeout when cabman was deleting rows older than 90 days. The second one was related to the batch file to start tabmon. As mentioned, the terminal was giving the error that the system could not find the specified file. We modified the original batch file and now it should be working fine. The issue can be closed.

Thanks!
